tensorflow/tools/ci_build/install/install_pip_packages.sh

```diff
@@ -15,11 +15,20 @@
 # ==============================================================================
 
 set -e
+ubuntu_version=$(cat /etc/issue | grep -i ubuntu | awk '{print $2}' | \
+  awk -F'.' '{print $1}')
 
 # Get the latest version of pip so it recognize manylinux2010
-easy_install3 -U pip
-easy_install -U pip
-
+# easy_install is not available on ubuntu 18.04
+if [[ "$ubuntu_version" == "18" ]]; then
+  wget https://bootstrap.pypa.io/get-pip.py
```

Comment:
Should we just switch all ubuntu versions to this?
This is the official installation method anyway.

That way, we can avoid version specific checks.

Reply:
I was trying to avoid changing existing behavior as much as possible, but I can test this on x86 and verify it works and then push an update to the PR.